x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.test.ts

```diff
@@ -5,7 +5,8 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { ReactElement } from 'react';
+import * as ReactIs from 'react-is';
```

Comment:
nit: if I'm not wrong we are trying to avoid this kind of imports. It should be:  
import { isFragment } from 'react-is';

Reply:
Make sense